src/main/java/org/tugraz/sysds/runtime/data/TensorBlock.java

```diff
@@ -100,7 +104,47 @@ public boolean isEmpty() {
 
 	public abstract double get(int r, int c);
 
+	/**
+	 * Set a cell to the value given as an `Object`. The type is inferred by either the `schema` or `valueType`, depending
+	 * if the `TensorBlock` is a `BasicTensor` or `DataTensor`.
+	 * @param ix indexes in each dimension, starting with 0
+	 * @param v value to set
+	 */
```

Comment:
Nice, thanks for keeping documentation on central data structures in mind.

Reply:
I tried to do it here and there, but didn't yet focus too much on it, since quite a few methods already changed their signature a few times. I will do a PR once we are happy with the general look and document all the central data structures.